README.md

```diff
@@ -1,5 +1,36 @@
-# asanaban
-Kanban on Asana
+# [Asanaban](https://asanaban.com/#!/setup)
+
+![Alt text](app/assets/images/screenshot.png?raw=true "Asanaban Demo")
+## Kanban on Asana
+
+Additional features beyond the Asana Kanban tool include:
+
+- Color-coded tasks
+- Limit to number of tasks per column before column turns red to warn you of too many tasks
+- Shows how many tasks are in a column
+- Multiple "Swimlanes" (rows) to sort tasks
+- Collapsible Columns
+- A variety of awesome background images to choose from 😎
+
+## Connection to Asana for the first time
+
+The first time you go to Asanaban, you'll be forwarded to a [setup page](https://asanaban.com/#!/setup) where you will be walked through a series of steps to connect your Asana account to Asanaban.
+
+## Syntax in Asana for storing Swimlanes, Columns, and column limits
+
+An Asanaban board will build itself from a specific syntax in the naming of Asana sections that looks like this:
+
+`swimlaneName:columnName|taskLimit`
```

Comment:
Do these need to be created by hand or does asanaban do that for you? Are these case sensitive? Is there an "order" to the sections (in asana) that matter? (whatever the answers, they should go into this part of the readme)